### ISSUE TITLE
Subsystem issues compendium

### ISSUE THREAD

**Issue by barbudreadmon** [libretro/RetroArch#9736]
I've seen dozens of people having problems with subsystem since i implemented it in fbalpha/fbneo. Afaik the feature is properly implemented in my code (see https://github.com/libretro/FBNeo/blob/d4545491ff09b5ea5d681ab140c4df603c892e4e/src/burner/libretro/libretro.cpp#L120-L144), and it actually works flawlessly from command-line on linux.
This core might be the only one using subsystems to load roms for other systems, a few months ago someone added an alternative way to load those other systems, based of folder names, as a workaround for all people having issues (see https://github.com/libretro/FBNeo/pull/159).

@twinaphex asked me to write some kind of compendium about the problems i know about, so that we could try leveraging this feature : 
1. subsystems aren't working properly with static linking platforms (https://github.com/libretro/RetroArch/issues/9735)
2. playlists feature doesn't handle subsystems (i heard it can if manually edited though)
3. at some point i think UI users were totally lost on how this feature works (way too many steps, and it didn't seem the same steps would work for everyone, it might have been related to 1. though), however @fr500 might have reduced the number of steps around 6 months ago (not sure since i never use UI).
4. i heard some menu drivers didn't support this, this information might be outdated though.

**Comment by fr500** [comment]
It also works fine from the GUI on non-static platforms.

Regarding the actual issues:

1. This seems to be the case on vita. Would be good to have testing on 
- [ ] switch
- [ ] wiiu
- [ ] xbone
- [ ] ps3
- [ ] vita
As the most complete static platforms. Maybe with another core (say tgb dual)

2. Playlists fully support subsystems, it's the scanner that doesn't
3. I don't know how to make submenus, this is the reason I implemented it like this.

I think it should work like this:

![image](https://user-images.githubusercontent.com/1721040/69014950-21173900-095d-11ea-9c57-ded399b49247.png)

![image](https://user-images.githubusercontent.com/1721040/69014958-35f3cc80-095d-11ea-9134-ebb4a5f6d007.png)

![image](https://user-images.githubusercontent.com/1721040/69014961-473cd900-095d-11ea-9e47-8332d3e26e4e.png)

4. Could be yeah

**Comment by twinaphex** [comment]
Yes, I'm going to submit myself to a console testing regime so we can get a good overview of what works and what doesn't on the various systems.
